### ISSUE TITLE
CGMES export: consider adding busbar sections for bus/breaker buses exported to node/breaker level

### ISSUE THREAD

**Issue by zamarrenolm** [powsybl/powsybl-core#2313]
* **Do you want to request a *feature* or report a *bug*?**
Feature

* **What is the current behavior?**
An IIDM bus/breaker voltage level may be exported to a node/breaker CGMES EQ profile.
For a reference, see example in `TopologyExportCornerCasesTest::testExportDisconnectedLoadBusBreaker`, where a bus/breaker topology with a bus that has two loads (one of them disconnected) is exported to CGMES 3 (CIM100). 

* **If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem**
After PR #2310 the mismatch in the definition of IIDM valid bus view buses from bus/breaker and node/breaker topologies makes the test fail because the network built as bus/breaker contains one bus view bus, and the network built from node/breaker does not contain any valid bus in the bus view.

* **What is the expected behavior?**
The bus view should contain the same number of bus view buses. 
Consider if it could be solved by adding busbar section objects to the CGMES EQ export for each configured bus in the bus/breaker view of IIDM.

* **What is the motivation / use case for changing the behavior?**
Uniform view of buses from bus/breaker and node/breaker network inputs.

* **Please tell us about your environment:**
  - PowSyBl Version: 4.11.0-SNAPSHOT with additional changes proposed in PR #2310

**Comment by miovd** [comment]
Yes, it is a good workaround to export in node-breaker. Have you tested that it works properly with this modification in export?
